torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -208,12 +208,64 @@ class TensorExprFuser {
   }
 
  private:
+  Node* aliasingSafeSubgraphMerge(
```

Comment:
Might be useful to have a comment explaining what the function does and what's the semantics of its arguments.